doc/development/api/virtual_schema_api.md

```diff
@@ -1211,7 +1204,7 @@ Notes:
 
 ##### LISTAGG
 
-`LISTAGG([DISTINCT] arg1[, separator] ON OVERFLOW {ERROR | TRUNCATE [truncationFiller] {WITH | WITHOUT} COUNT}) [WITHIN GROUP (orderBy)]`
+`LISTAGG([DISTINCT] arg[, separator] ON OVERFLOW {ERROR | TRUNCATE [truncationFiller] {WITH | WITHOUT} COUNT}) [WITHIN GROUP (orderBy)]`
```

Comment:
To match GROUP_CONCAT and because we only allow string literals here:
```suggestion
`LISTAGG([DISTINCT] arg[, 'separator'] ON OVERFLOW {ERROR | TRUNCATE ['truncationFiller'] {WITH | WITHOUT} COUNT}) [WITHIN GROUP (orderBy)]`
```